src/actions/allActivitiesActions.js

```diff
@@ -0,0 +1,57 @@
+import axios from 'axios';
```

Comment:
The Naming of the file is a bit  misleading maybe we can rename them to `getActivities` `createActivities` 
if see allActivitiesActions I think all the actions for activities are contained in the file.